plugins/in_systemd/systemd.c

```diff
@@ -276,15 +282,28 @@ static int in_systemd_collect(struct flb_input_instance *i_ins,
     if (ret_j == 0) {
         ctx->pending_records = FLB_FALSE;
         return FLB_SYSTEMD_OK;
-    }
-
+    } else if (ret_j > 0) {
```

Comment:
Please add the "else if" statement in a new line, according to:

https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md